### PR TITLE
Fix description text area when creating a new team [SCI-9138]

### PR DESCRIPTION
**Pull request by sboursen-scinote**
Jira ticket: [SCI-9138](https://scinote.atlassian.net/browse/SCI-9138)

### What was done
Fix the description text area when creating a new team

[SCI-9138]: https://scinote.atlassian.net/browse/SCI-9138?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/views/users/settings/teams/new.html.erb [hunk 1]
```diff
@@ -18,7 +18,7 @@
       </div>
 
       <div class="form-group" style="max-width: 500px;">
-        <%= f.text_area :description, label: t('users.settings.teams.new.description_label') %>
+        <%= f.text_area :description, class: "form-control", label: t('users.settings.teams.new.description_label') %>
```
Comment (aignatov-bio) [inline app/views/users/settings/teams/new.html.erb:21]: Does it must be 'sci-input-field' ?

**Review by aignatov-bio — CHANGES_REQUESTED** [review]

File: app/views/users/settings/teams/new.html.erb (hunk 1 above)
Comment (aignatov-bio) [reply app/views/users/settings/teams/new.html.erb]: you need add `sci-input-container` instead of form-group

Comment (sboursen-scinote) [reply app/views/users/settings/teams/new.html.erb]: It breaks the layout as sci-input-container are inline-box

Comment (aignatov-bio) [reply app/views/users/settings/teams/new.html.erb]: And what problem to fix layout?

**Review by aignatov-bio — APPROVED** [review]

**Pull request merged**